css/index.css

```diff
@@ -1,163 +1,267 @@
-*{
-    margin: 0; /*from youtube*/
-    padding:0;
+/* Smartphones (portrait and landscape) */
+@media only screen and (max-width: 600px) {
```

Comment:
Not sure these media queries are given the desired effect because order matters in the css file,  the last rule applicable rule "wins" so for the most part the media queries in the front of the file are overidden by the queries later in the file - for example your background colors are always overidden